torch/onnx/utils.py

```diff
@@ -1776,12 +1802,6 @@ def _run_symbolic_function(
                 attrs = {
                     k: symbolic_helper._node_get(node, k) for k in node.attributeNames()
                 }
-                if _need_symbolic_context(symbolic_fn):
-                    # TODO(justinchuby): Refactor how we check for the need of the symbolic context
-                    ctx = _exporter_states.SymbolicContext(
-                        _params_dict, env, node, block
-                    )
-                    return symbolic_fn(ctx, graph_context, *inputs, **attrs)
                 # PythonOp symbolic need access to the node to resolve the name conflict,
                 # this is inconsistent with regular op symbolic.
                 if op_name == "PythonOp":
```

Comment:
What happens if a PythonOp needs symbolic_context? @BowenBao how can we handle python ops elegantly?

Reply:
We can remove this.

Reply:
Removed